docs/source-pytorch/common/lightning_module.rst

```diff
@@ -85,38 +85,42 @@ Here are the only required methods.
 .. code-block:: python
 
     import lightning.pytorch as pl
-    import torch.nn as nn
-    import torch.nn.functional as F
+    import torch
 
+    from lightning.pytorch.demos import Transformer
 
-    class LitModel(pl.LightningModule):
-        def __init__(self):
+
+    class LightningTransformer(pl.LightningModule):
+        def __init__(self, vocab_size):
             super().__init__()
-            self.l1 = nn.Linear(28 * 28, 10)
+            self.model = Transformer(vocab_size=vocab_size)
 
-        def forward(self, x):
-            return torch.relu(self.l1(x.view(x.size(0), -1)))
+        def forward(self, inputs, target):
+            return self.model(inputs, target)
 
         def training_step(self, batch, batch_idx):
-            x, y = batch
-            y_hat = self(x)
-            loss = F.cross_entropy(y_hat, y)
+            inputs, target = batch
+            output = self(inputs, target)
+            loss = torch.nn.functional.nll_loss(output, target.view(-1))
             return loss
 
         def configure_optimizers(self):
-            return torch.optim.Adam(self.parameters(), lr=0.02)
+            return torch.optim.SGD(self.model.parameters(), lr=0.1)
 
 Which you can train by doing:
 
 .. code-block:: python
 
-    train_loader = DataLoader(MNIST(os.getcwd(), download=True, transform=transforms.ToTensor()))
-    trainer = pl.Trainer(max_epochs=1)
-    model = LitModel()
+    from torch.utils.data import DataLoader
+
+    dataset = pl.demos.WikiText2()
```

Comment:
Here too

Reply:
this is changed in [6671aa6](https://github.com/Lightning-AI/lightning/pull/18452/commits/6671aa6215041d1f8598854e39fd7a5c5330cb69)